src/js/oncoprintwebglcellview.ts

```diff
@@ -1114,6 +1114,7 @@ export default class OncoprintWebGLCellView {
             this.computeVertexPositionsAndVertexColors(model, track_ids[i]);
             this.computeVertexColumns(model, track_ids[i]);
         }
+        this.setTrackShowGaps(model);
```

Comment:
The way that the existing code is written is not ideal. All these single purpose callbacks is very confusing, so I'm sorry for that. But we wouldn't want to do this - it would be expensive to force reloading the shaders every time, and theres a redundant call to renderAllTracks.